backend/src/api/routes/applicationRoute.ts

```diff
@@ -9,8 +9,7 @@ router
   .get('/', run((req: Request) => getApps(req.user as ITransportedUser)))
   .get('/:id', run((req: Request) => getAppById(req.params.id)))
   .post('/', run((req: Request) => addApp(req.body, req.user as ITransportedUser)))
-  .put('/:id', run((req: Request) => (
-    updateApp(req.params.id, req.body.name, req.user as ITransportedUser))))
+  .put('/:id', run((req: Request) => (updateApp(req.params.id, req.body.name, req.user as ITransportedUser))))
```

Comment:
We don't need to set `req: Request` explicitly since `req` already has its type set implicitly. Could you please fix it for all app routes?
```suggestion
  .put('/:id', run(req => (updateApp(req.params.id, req.body.name, req.user as ITransportedUser))))
```